openpaas-james/apps/distributed-es6-backport/src/main/conf/jmap.properties

```diff
@@ -0,0 +1,18 @@
+# Configuration file for JMAP
+# Read https://james.apache.org/server/config-jmap.html for further details
+
+enabled=true
+
+tls.keystoreURL=file://conf/keystore
+tls.secret=james72laBalle
```

Comment:
We don't need JMAP TLS config

Reply:
Why not?

Reply:
Need this for keystore JWT generation

Reply:
A longer answer is that it is needed by JMAP draft to sign continuation tokens. Legacy...